src/raftstore/store/store.rs

```diff
@@ -403,7 +422,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
     fn on_raft_base_tick(&mut self, event_loop: &mut EventLoop<Self>) {
         let timer = self.raft_metrics.process_tick.start_timer();
         for (&region_id, peer) in &mut self.region_peers {
-            if !peer.get_store().is_applying() {
+            // When having pending snapshot, if election timeout is met, it can't pass
+            // the pending conf change check because first index has been updated to
+            // a value that is larger than last index.
+            self.pending_raft_groups.insert(region_id);
```

Comment:
the optimization in #1619 here seems needless if we add pending_raft_groups every time?